### ISSUE TITLE
How to collapse RibbonGroups with an InRibbonGallery

### ISSUE THREAD

**Issue by pschimmel** [fluentribbon/Fluent.Ribbon#897]
Hi!

I've added ribbon galleries to groups, using Fluent 8.0.3. The resizing of the galleries works fine. However, I the RibbonGroup does not collapse to a dropdown button:

![Gall](https://user-images.githubusercontent.com/9028723/106322161-b1495180-6275-11eb-9e0c-9e09166f5a91.gif)

My assumption is that there is something wrong with the ReduceOrder I defined in XAML, but I cannot get it to behave correctly:

```xml
<fluent:RibbonTabItem Header="Tab1" ReduceOrder="FirstGalleryGroup,FirstGalleryGroup,FirstGalleryGroup,(SecondGalleryGroup),(SecondGalleryGroup),(SecondGalleryGroup),(ThirdGalleryGroup),(ThirdGalleryGroup),(ThirdGalleryGroup)">
  <fluent:RibbonGroupBox Name="FirstGalleryGroup" Header="Group1" ItemsSource="{Binding Group1Items}" />
  <fluent:RibbonGroupBox Name="SecondGalleryGroup" Header="Group2" ItemsSource="{Binding Group2Items}" />
  <fluent:RibbonGroupBox Name="ThirdGalleryGroup" Header="Group3" ItemsSource="{Binding Group3Items}" />
</fluent:RibbonTabItem>
```

Here's my small example project:

[FluentTest.zip](https://github.com/fluentribbon/Fluent.Ribbon/files/5896104/FluentTest.zip)

Can you give me a hint what's wrong?

**Comment by batzen** [comment]
`()` is for scale. So you basically only allowed state changes for the first group 😉

**Comment by pschimmel** [comment]
Amazing. Thank you!

**Comment by pschimmel** [comment]
Hi @batzen!
I'm afraid I found another bug in the context of InRibbonGalleries and resizing.
With your hint above, I managed to activate correct resizing of the galleries, also resizing of the dropdown buttons.
However when I reduce the size even more that the group which contains the gallery reduces to a dropdown button and I try to open that dropdown, the application freezes.
Here's a video and my sample application:
![Frozen](https://user-images.githubusercontent.com/9028723/106907749-e1be3f00-66fe-11eb-955f-74f7addccf9b.gif)
[GalleryTests.zip](https://github.com/fluentribbon/Fluent.Ribbon/files/5926396/GalleryTests.zip)
